<launch>
  <param name="opcua_client_node" value="/opcua/opcua_client" />
  <group ns="transport_raws">
    <node pkg="robonomics_game_transport" name="stacker_raw" type="stacker_node.py">
      <param name="opcua_server_namespace" value="ns=4;s=/Airalab/Stacker_raw" />
      <param name="direction" value="forward" />
      <param name="timeout" value="5000" />
    </node>
  </group>
  <group ns="transport_goods">
    <node pkg="robonomics_game_transport" name="stacker_goods" type="stacker_node.py">
      <param name="opcua_server_namespace" value="ns=4;s=/Airalab/Stacker_goods" />
      <param name="direction" value="backward" />
      <param name="timeout" value="5000" />
    </node>
    <node pkg="robonomics_game_transport" name="conveyor" type="conveyor_node.py">
      <param name="opcua_server_namespace" value="ns=4;s=/Airalab/Conveyor" />
      <param name="tr_load_time" value="2000" />
      <param name="tr_unload_time" value="2000" />
      <param name="tr_timeout" value="5000" />
      <param name="t;_unload_time" value="2000" />
    </node>
    <node pkg="robonomics_game_transport" name="sensor" type="color_sensor_node.py">
      <param name="opcua_server_namespace" value="ns=4;s=/Airalab/Sensor" />
    </node>
  </group>
</launch>
